spring-kafka/src/main/java/org/springframework/kafka/config/StreamsBuilderFactoryBean.java

```diff
@@ -188,6 +188,7 @@ public void setStateListener(KafkaStreams.StateListener stateListener) {
 		this.stateListener = stateListener; // NOSONAR (sync)
 	}
 
+	@SuppressWarnings("deprecation")
```

Comment:
I don't see that you have change `kafka-streams` version, but probably this is a missed artifact from the previous project state. I wonder what is a replacement for this deprecated API then...

Reply:
I haven't changed any versions; this is just for compatibility, to allow users to override the client version.

kafka-streams has to match kafka-clients version.

```java
     * @deprecated Since 2.8.0. Use {@link KafkaStreams#setUncaughtExceptionHandler(StreamsUncaughtExceptionHandler)} instead.
```

Reply:
Oh; I see what you mean; we really don't need these suppressions because we're still compiling against the 2.7 clients. Will revert.